2022-02-14 16:56:46 [INFO] 获得列名:attach====成功 [case.py:14]
2022-02-14 16:56:46 [INFO] 获取入口成功，服务器：debug；数据库：debug [conf.py:10]
2022-02-14 16:56:46 [INFO] 获取服务器信息成功:http://192.168.222.154:80 [conf.py:20]
2022-02-14 16:56:46 [INFO] 读取excel文件:../excelcases/attach.xlsx====成功 [case.py:34]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:56:47 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:56:47 [INFO] 存入公共字典,字典：{'name1': 'admin', 'passwd': '123456'}====成功 [case.py:67]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] login_01:测试成功登录====响应结果比对====成功! [case.py:104]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:56:47 [INFO] 存入公共字典,字典：{'name1': 'admin', 'passwd': '123456', 'name2': 'dd'}====成功 [case.py:67]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] login_02:测试获取用户信息====响应结果比对====成功! [case.py:104]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] 无可存入公共字典的信息,字典：{'name1': 'admin', 'passwd': '123456', 'name2': 'dd'} [case.py:59]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [WARNING] login_03:测试登录失败====响应结果比对====失败!====expect:测试登录失败====real:﻿用户test11登录验证成功<a href='/apitest/refer/info/'>查看个人信息</a> [case.py:107]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [INFO] 无可存入公共字典的信息,字典：{'name1': 'admin', 'passwd': '123456', 'name2': 'dd'} [case.py:59]
2022-02-14 16:56:47 [INFO] 发送请求====成功 [case.py:87]
2022-02-14 16:56:47 [WARNING] login_04:测试获取用户信息失败====响应结果比对====失败!====expect:测试获取用户信息失败====real:悄悄告诉你！你的用户名是test11，密码是123456 [case.py:107]
